libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -225,6 +225,19 @@ void GCS_MAVLINK::send_power_status(void)
                                   hal.analogin->power_status_flags());
 }
 
+#if AP_SCHEDULER_ENABLED
+// cap the MAVLink message rate. It can't be greater than 0.8 * SCHED_LOOP_RATE
+uint16_t GCS_MAVLINK::cap_message_interval(uint16_t interval_ms) const {
```

Comment:
nitpick: normally we put the curly brace on its own line

Reply:
Fixed